compose.test.watch.yaml

```diff
@@ -2,7 +2,6 @@ services:
   fcp-sfd-frontend:
     command: npm run test:watch
     volumes:
-      - ./.git:/home/node/.git
```

Comment:
Binding `.git` directory was a pre-requisite of Jest for file watching.  Vitest file watching is a bit "smarter" so this is not needed.